Framework/src/CustomParameters.cxx

```diff
@@ -202,7 +202,20 @@ void CustomParameters::populateCustomParameters(const boost::property_tree::ptre
   for (const auto& [runtype, subTreeRunType] : tree) {
     for (const auto& [beamtype, subTreeBeamType] : subTreeRunType) {
       for (const auto& [key, value] : subTreeBeamType) {
-        set(key, value.get_value<std::string>(), runtype, beamtype);
+        // Check if value has children (thus it is json)
+        if (value.empty()) { // just a simple value
```

Comment:
I could not believe that `empty()` checks if there are children, but indeed it does... https://github.com/boostorg/property_tree/blob/2669458cebee100bc1f9f4a9d2f0cd95d07740bc/include/boost/property_tree/ptree.hpp#L107